### PR TITLE
Description formula

### PR DESCRIPTION
**Pull request by nobuti**
This PR implements #11469.

### CODE REVIEW

**Comment by xavijam** [comment]
Please, update NEWs.md 💋 .

**Comment by nobuti** [comment]
Do you mean how much code we could extract to a middle component? I don't know really, we are augmenting the base component `Backbone.Form.editors.Text` and `Backbone.Form.editors.TextArea` from Backbone.Forms. I guess this would be a nice case for a mixin, but not completely sure.

**Comment by matallo** [comment]
I mean, looking at both I thought they could be consolidated, but taking a look back at backbone-forms it is like that for a reason https://github.com/powmedia/backbone-forms/blob/master/src/editors/textarea.js#L9

sorry for the hassle 👍 👍 👍

**Comment by javitonino** [comment]
Found a couple of issues:

In time series, the fill color selector no longer has 100% width.
screenshot_20170220_122849

In all widgets, the subtitle under "3 Behaviour" has changed and now it says description. It should still say "DEFINE HOW YOUR WIDGET INTERACTS WITH THE DATA" according to the design screenshots.
screenshot_20170220_122908

**Comment by matallo** [comment]
very well spotted @javitonino I fixed the width of the form, now working on changing the literal for that specific case and revert the former literal in the other widgets

**Comment by javierarce** [comment]
### Acceptance

It looks like the subtitle of the section wasn't correctly changed:

screen shot 2017-02-21 at 11 59 05

**Comment by matallo** [comment]
If I recall correctly, that literal is "Description" indeed, it is in the rest of the types which remains the former literals

**Comment by javierarce** [comment]
But it shouldn't right? @javitonino mentioned it [at the end of this comment](https://github.com/CartoDB/cartodb/pull/11575#issuecomment-281056033):

> In all widgets, the subtitle under "3 Behaviour" has changed and now it says description. It should still say "DEFINE HOW YOUR WIDGET INTERACTS WITH THE DATA" according to the design screenshots.

**Comment by matallo** [comment]
Oh, I must have misunderstood then, derived from the changes in this commit https://github.com/CartoDB/cartodb/pull/11575/commits/95e2cb82a5c898ee244cbd193e5032f2060240cc#diff-75426b988ee7e728691a6ee313a35c78L2062

I thought the description for that type was "Description"

summoning @CartoDB/design to clarify

**Comment by javierarce** [comment]
No problem!

Also, the design in the ticket description shows the following header:

screen shot 2017-02-21 at 12 50 07

while this ticket shows

screen shot 2017-02-21 at 12 49 48

Maybe @CartoDB/design can clarify that too.

**Comment by arianaescobar** [comment]
Clarifying: 

- The subtitle of Step 3 is: "DEFINE HOW YOUR WIDGET INTERACTS WITH THE DATA"
- Regarding the headers, I opened another ticket for that (#11468). The correct one has the type name on the step title and a brief description of each widget on the subtitle. 

Also...

Could we align the "Description" label and checkbox to the top, instead of being aligned to the middle, so it looks like on the designs? 

screen shot 2017-02-21 at 13 25 18

**Comment by nobuti** [comment]
This change is going to affect all forms, we should it carefully following the instructions here: https://github.com/CartoDB/cartodb/issues/11469#issuecomment-281355003

**Comment by piensaenpixel** [comment]
yes, it's a bit frail

**Comment by ivanmalagon** [comment]
Last fixes

1. Description label is aligned also in textarea. @piensaenpixel helped with the CSS. The changes affect only to check inputs in forms but everything remains the same. You can see in the screenshot below that the other two checks are not misaligned.

1. The description of step 3 goes back to its former text.

formula

File: app/assets/stylesheets/editor-3/forms.css.scss [hunk 1]
```diff
@@ -78,13 +78,17 @@
 
 .Editor-checker {
   width: 100%;
+  @include align-items(flex-start);
 }
 .Editor-checker--slim .Editor-formInner {
   min-height: 20px;
 }
 .Editor-checkerInput {
```
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/forms.css.scss]: I think we could add this fix also for CDB-Legend, this way we'd leave this fixed for the future, what do you think?

Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/forms.css.scss]: ok, na forget it :)

**Comment by ivanmalagon** [comment]
Accepted but waiting for all the checks to pass.

In local, all test are passing but travis and default have failed.

**Comment by ivanmalagon** [comment]
retest this please

**Comment by ivanmalagon** [comment]
Still failing. Investigating. 🔍

**Comment by ivanmalagon** [comment]
Hi @CartoDB/frontend 

I ran into a big problem with `npm-shrinkwrap`. Running `update-internal-deps` with a newer version of node mess that file. To the point of not being recognised even by its mother.

I was using v6.9.2 Moving back to v0.10.48 cleaned all the mess.

Just saying this to advice you to not making the same mistakes as me. :man_facepalming:

**Comment by nobuti** [comment]
a .nvmrc file should be great in this case.

**Comment by xavijam** [comment]
Agree with @nobuti.

**Comment by ivanmalagon** [comment]
Deployed to produ.

**Pull request merged**